src/verifier/client.rs

```diff
@@ -43,17 +43,18 @@ impl DIDClient {
     pub async fn new(
         vm: String,
         signer: Arc<dyn RequestSigner + Send + Sync>,
-        resolver: &dyn DIDResolver,
+        resolver: impl JWKResolver,
```

Comment:
This may be incorrect resolver type. 

I'm assuming the verification method type needs to resolve into a vm-specific key type, not necessarily a JWK.